src/Materials/Textures/imageCubeTexture.ts

```diff
@@ -0,0 +1,194 @@
+import { PanoramaToCubeMapTools } from '../../Misc/HighDynamicRange/panoramaToCubemap';
+import { Engine } from "../../Engines/engine";
+import { BaseTexture } from '../../Materials/Textures/baseTexture';
+import { Texture } from '../../Materials/Textures/texture';
+import { Scene } from "../../scene";
+import { Nullable } from "../../types";
+
+/**
+ * This represents a texture coming from an image supported by the web browser canvas.

```

Comment:
Could you be more explicit about the need for equirectangular data?

Reply:
Can you elaborate a bit more what you mean by that? I'm guessing you either mean the docstring should be more descriptive or why this addition is needed?

Reply:
I meant for the comment itself. We should say something like:
/**
* This represents a texture coming from an equirectangular image supported by the web browser canvas.

Reply:
Docstring is now updated.